Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
I don't think I was suggesting installing the built products of the embedding example. The sources and setup for recreating them for testing, yes.